modules/nf-core/gatk4/germlinecnvcaller/meta.yml

```
@@ -0,0 +1,56 @@
+name: "gatk4_germlinecnvcaller"
+description: Calls copy-number variants in germline samples given their counts and the output of DetermineGermlineContigPloidy.
+keywords:
+  - gatk
+  - gatk4_germlinecnvcaller
+tools:
+  - "gatk4":
+      description:
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/categories/360002369672s
+      doi: "10.1158/1538-7445.AM2017-3590"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - tsv:
+      type: file
+      description: TSV file
+      pattern: "*.tsv"
+  - bed:
```

```suggestion
  - intervals:
```
This is the intervals file right?